### ISSUE TITLE
Databases cleaning

### ISSUE THREAD

**Issue by mrngg** [CIRCL/AIL-framework#368]
Hi there,

I have limited disk capacity and I would like to cron a daily cleaning of pastes, dumps, etc but without removing users. It seems like `reset_AIL.sh` was meant for this purpose but it is not up-to-date as some directories are missing or had their names changed.

Any help would be greatly appreciated, many thanks.

**Comment by Terrtia** [comment]
Hi @mrngg 

``reset_AIL.sh`` should be fixed with 574b64b33b2d52a4c2ed209f9258a685d5747474

You can use the --softReset option to keep all user accound

**Comment by mrngg** [comment]
Hi back @Terrtia,

Thank you for this improvement. 

I did not test the new `reset_AIL.sh` script until yesterday and I found out that running it with the `--softReset` option still deletes users as I cannot pass the login screen. Nonetheless, I might have figured out why it does not work. Actually, lines 26 to 34 seem unwanted as the if statement compares to empty strings and the code block inside it is redundant to the block starting at line 129 which is processed when running a regular/hard reset. Therefore, running the script by commenting out lines 26 to 34 solved my issue.
